### PR TITLE
Issue #637 Extend allocated time for integration tests to 90m

### PR DESCRIPTION
**Pull request by jsliacan**
90min should suffice for now

### CODE REVIEW

**Comment by praveenkumar** [comment]
@jsliacan Can we put something on the commit log why we increase this limit like which test we added due to that this time is increased ?

**Comment by jsliacan** [comment]
@praveenkumar changed the commit description as you suggested, plus the #637 description links to a particular instance when timeout happened.

**Comment by praveenkumar** [comment]
retest this please.

File: Makefile [hunk 1]
```diff
@@ -124,7 +124,7 @@ integration: GODOG_OPTS = --godog.tags=$(GOOS)
 integration:
 	@$(call check_defined, BUNDLE_LOCATION, "'make integration' requires BUNDLE_LOCATION to contain the full path to a bundle file")
 	@$(call check_defined, PULL_SECRET_FILE, "'make integration' requires PULL_SECRET_FILE to point to a file with the pull secret to use")
-	@go test --timeout=60m $(REPOPATH)/test/integration -v --tags=integration $(GODOG_OPTS) $(BUNDLE_LOCATION) $(PULL_SECRET_FILE)
+	@go test --timeout=90m $(REPOPATH)/test/integration -v --tags=integration $(GODOG_OPTS) $(BUNDLE_LOCATION) $(PULL_SECRET_FILE)
```
Comment (praveenkumar) [inline Makefile]: :+1:

**Pull request merged**